File: src/main/java/org/projecttl/api/inventorygui/InventoryGUI.java
package org.projecttl.api.inventorygui;

import org.bukkit.Bukkit;
import org.bukkit.ChatColor;
import org.bukkit.configuration.InvalidConfigurationException;
import org.bukkit.plugin.java.JavaPlugin;

import java.io.File;
import java.io.IOException;
import java.util.Objects;

public class InventoryGUI extends JavaPlugin {
    private static InventoryGUI INSTANCE;

    @Override
    public void onEnable() {
        INSTANCE = this;
        getLogger().info("<InventoryGUI> " + ChatColor.GREEN + "API has successful enabled.");

        Bukkit.getPluginManager().registerEvents(new Listener(), this);
    }

    @Override
    public void onDisable() {
        getLogger().info("<InventoryGUI> " + ChatColor.DARK_RED + "API has successful disabled.");
    }

    public static InventoryGUI getInstance() {
        return INSTANCE;
    }
}

File: src/main/kotlin/net/projecttl/api/inventorygui/InventoryGUI.kt
package net.projecttl.api.inventorygui

import org.bukkit.ChatColor
import org.bukkit.plugin.java.JavaPlugin

class InventoryGUI: JavaPlugin() {

    override fun onEnable() {
        logger.info("<InventoryGUI> ${ChatColor.GREEN}InventoryGUI API is successful loaded!")
    }

    override fun onDisable() {
        logger.info("<InventoryGUI> ${ChatColor.GREEN}InventoryGUI API is successful unloaded!")
    }
}